packages/e2e-tests/helpers/console-panel.ts

```diff
@@ -123,8 +123,9 @@ export function findConsoleMessage(
     ? `[data-test-message-type="${messageType}"]`
     : '[data-test-name="Message"]';
 
+  // Use single quotes because a couple messages have double quoted text displayed
```

Comment:
Wonder if we should throw if you pass a string with double quotes? (Or we could switch outer quote styles, depending?)